CK.DB.Acl/Res/Model/CK.AclConfigMemoryTable.Settle.1.0.1.to.1.0.2.sql

```diff
@@ -0,0 +1,19 @@
+declare @ActorId int;
+declare @AclId int;
+declare @KeyReason varchar(128) = '';
+
+declare InvalidConfigsCursor cursor local static read_only forward_only
```

Comment:
Simply use local fast_forward. Below a typical example: 

```
	declare @UserId int;
	declare @CUser cursor;
	set @CUser = cursor local fast_forward for 
		select ActorId from CK.tActorProfile p 
						inner join CK.tUser u on u.UserId = p.ActorId
						where p.GroupId = @GroupId and p.ActorId <> @GroupId;
	open @CUser;
	fetch from @CUser into @UserId;
	while @@FETCH_STATUS = 0
	begin
		exec CK.sGroupUserRemove @ActorId, @GroupId, @UserId;
		fetch next from @CUser into @UserId;
	end
	deallocate @CUser;
```

See: https://stackoverflow.com/questions/2280452/what-is-the-advantage-of-using-fast-forward-for-defining-a-cursor